README.md

```diff
@@ -165,3 +165,29 @@ To run style check manually, simply run the following from the top level folder.
 ```
 flake8 pyclaragenomics/
 ```
+
+## Running CI Tests Locally
+Requirements:
+1. docker (https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+2. nvidia-docker (https://github.com/NVIDIA/nvidia-docker)
+3. nvidia-container-runtime (https://github.com/NVIDIA/nvidia-container-runtime)
+
+Run the following command to execute the CI build steps inside a container locally:
+```bash
+bash ci/local/build.sh -r <ClaraGenomicsAnalysis repo path>
+```
+ci/local/build.sh script was adapted from [rapidsai/cudf](https://github.com/rapidsai/cudf/tree/branch-0.11/ci/local)
+
+The default docker image is **clara-genomics-base:cuda10.0-ubuntu16.04-gcc5-py3.7**.
+Other images from [gpuci/clara-genomics-base](https://hub.docker.com/r/gpuci/clara-genomics-base/tags) repository can be used instead, by using -i argument
+```bash
+bash ci/local/build.sh -r <ClaraGenomicsAnalysis repo path> -i clara-genomics-base:cuda10.0-ubuntu18.04-gcc7-py3.6
+```  
+
+Please note, your git repository will be mounted to the container, any untracked files will be removed from it. 
```

Comment:
Let's move this note to the beginning of this section :)

Reply:
You are right!